src/WorldStorage/WSSAnvil.cpp

```diff
@@ -2011,6 +2014,212 @@ void cWSSAnvil::LoadPaintingFromNBT(cEntityList & a_Entities, const cParsedNBT &
 
 
 
+void cWSSAnvil::LoadArmorStandFromNBT(cEntityList & a_Entities, const cParsedNBT & a_NBT, int a_TagIdx)
+{
+	std::unique_ptr<cArmorStand> ArmorStand = cpp14::make_unique<cArmorStand>(Vector3d(), 0.0);
+	if (!LoadEntityBaseFromNBT(*ArmorStand.get(), a_NBT, a_TagIdx))
+	{
+		return;
+	}
+
+
+	/**
+		Basic properties
+	*/
```

Comment:
Please don't use doxy-comments in regular code. Use a normal single-line `// Comment`